services/app-web/src/components/FileUpload/FileUpload.tsx

```diff
@@ -329,7 +353,16 @@ export const FileUpload = ({
                 </span>
             )}
 
-            <h5 tabIndex={-1} ref={summaryRef} className="text-normal font-body-sm margin-0">{summary}</h5>
+            <div style={{ display: 'flex' }}>
+                <h5
+                    tabIndex={-1}
+                    ref={summaryRef}
+                    className="text-normal font-body-sm margin-0"
+                >
```

Comment:
@jim This is my new idea. Keep the H5 only on "the X files added" text. Move the rest of the summary text (`summaryDetailText`) outside. That way its not `aria-hidden` but its also on the page. Thoughts?